db/User.js

```diff
@@ -141,5 +141,40 @@ User.findByToken = async function findByToken(token) {
     throw error;
   }
 };
-
+User.findByAdminToken = async function(token){
+  try{
+    let user = await this.findByToken(token);
+    if(user.isAdmin){
+      return user;
+    }
+    throw 'error';
+  }
+  catch(ex){
+    const error = new Error("bad token");
+    error.status = 401;
+    throw error
+  }
+};
+User.prototype.createProduct = async function(productReq){
+  const product = await conn.models.product.Create(productReq);
+  return product;
+};
+User.prototype.updateProduct = async function(productReq, id){
+  let product = await conn.models.product.findByPk(id*1);
+  // product = await conn.models.product.update({
+  //   name: productReq.name,
+  //   description: productReq.description,
+  //   ml: productReq.ml,
+  //   price: productReq.price,
+  //   limit: productReq.limit,
+  //   imgUrl: productReq.imgUrl
+  // });
```

Comment:
Let's clean up commented out code. Good job in using the `productReq` for the update. A lot cleaner